ui/src/App.js

```diff
@@ -107,15 +107,15 @@ class App extends Component {
     }
 
     serializeFilters(searchFilters) {
-        let filterStr = [];
+        let filterList = [];
         searchFilters.forEach((values, key) => {
             if (values.length > 0) {
```

Comment:
Throw an error if values.length == 0, or don't check

Reply:
This is not an error case, check > uncheck will result in an initialized key corresponding to an empty array, in which case we'll just ignore it rather than provide an empty filter.